compiler/aarch64/codegen/OpBinary.cpp

```diff
@@ -389,6 +389,8 @@ const OMR::ARM64::InstOpCode::OpCodeBinaryEntry OMR::ARM64::InstOpCode::binaryEn
 		0x9BA08000,	/* UMSUBL    	umsubl	 */
 		0x9B400000,	/* SMULH     	smulh	 */
 		0x9BC00000,	/* UMULH     	umulh	 */
+		0X1F000000, /* FMADD        fmaddd   */
+		0X1F400000, /* FMADD        fmadds   */
```

Comment:
The alignment of the comments looks wrong here.

Reply:
The file is using tab instead of space. Fixed.